Sources/Tracing/TracerProtocol+Legacy.swift

```diff
@@ -69,12 +69,12 @@ public protocol LegacyTracerProtocol: InstrumentProtocol {
 // MARK: Legacy implementations for Swift 5.7
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *) // for TaskLocal Baggage
-extension LegacyTracerProtocol {
+extension LegacyTracer {
     // ==== startSpan ---------------------------------------------------------
 
     /// Start a new span returning an existential ``Span`` reference.
     ///
-    /// - Warning: This method will be deprecated in favor of `TracerProtocol/withSpan` as soon as this project is able to require Swift 5.7.
+    /// - Warning: This method will be deprecated in favor of `Tracer/withSpan` as soon as this project is able to require Swift 5.7.
```

Comment:
We need <code>\`\`Tracer/withSpan\`\`</code> here to correctly refer to the function.

Reply:
Yeah there's a whole "update all the docs" PR that will handle all updates, thanks for pointing it out tho!